impl/src/main/java/com/okta/sdk/impl/oauth2/AccessTokenRetrieverServiceImpl.java

```diff
@@ -60,6 +59,7 @@
     private static final Logger log = LoggerFactory.getLogger(AccessTokenRetrieverServiceImpl.class);
 
     private static final String TOKEN_URI  = "/oauth2/v1/token";
+    private static final int DEFAULT_EXPIRATION_OFFSET = 3600;
```

Comment:
The default is set twice (once here and once in the okta.properties file), idealy set it only in the okta.props file to reduce the chance of error in the future